content/en/contribute/medic/product-development-process/ux-research-repo/adding-nuggets.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Adding Nuggets"
+linkTitle: "Adding Nuggets"
+weight: 3
+description: >
+  How to add nuggets in the UX Research Repository
+---
+
+The majority of research observations will come from interviews with CHT users. Here’s how you will enter these observations, called “Nuggets”, into the UX Research Repo.
+
+After conducting your interview, copy the video recording to Google Drive (more info [here]({{% ref "about-interviews" %}})). You’ll need to reference this later. 
+
+## High Level Steps
+
+1. Add the **Organization** (*of the person you interviewed*) if it doesn’t already exist
+2. Create the **Interview Subject** if this is the first time we’ve interviewed or entered observations from this person**.**
+3. Create a record for the **Sample** of the recording.
+4. Add research observations a.k.a. “**Nuggets”!** 
+
+Ready to try? See a [demo](https://drive.google.com/file/d/1YPXoba9gVmD7SP-X88PpJIsIVGvY86_G/view?usp=share_link) or check out the [Quick Links]({{% ref "ux-research-repo#quick-links" %}}) and give it a go!
```

Comment:
We haven't actually published "quick links" anywhere on the linked/ref page, and we don't really want to make them public since some of the data entry pages include drop downs with PII.  On the UX Research Repo page we say... 

> "To gain access to the UX Research Repo, please reach out in the #product-owner-chatter Slack channel. The access to the repository is currently restricted to the Medic team." 

Perhaps we can say something similar here?

Reply:
Great find, thanks @michaelkohn !